src/boot/ksdk1.1.0/devAMG8834.c

```diff
@@ -56,68 +56,72 @@
 #include "SEGGER_RTT.h"
 #include "warp.h"
 
-
-
-extern volatile WarpI2CDeviceState	deviceAMG8834State;
-extern volatile uint32_t		gWarpI2cBaudRateKbps;
-extern volatile uint32_t		gWarpI2cTimeoutMilliseconds;
-extern volatile uint32_t		gWarpSupplySettlingDelayMilliseconds;
-
+extern volatile WarpI2CDeviceState deviceAMG8834State;
```

Comment:
This doesn't match the alignment convention (whether we agree on tabs versus spaces or not).

Reply:
I have revered this back.